### ISSUE TITLE
Icon Animation

### ISSUE THREAD

**Issue by naher94** [naidurubini/empact.cc#4]
https://maxwellito.github.io/vivus/

**Comment by naidurubini** [comment]
Yes!!

**Comment by naidurubini** [comment]
![icons](https://user-images.githubusercontent.com/36688762/36746964-a264ecc2-1bc1-11e8-9de2-117b2f90c362.png)

**Comment by naidurubini** [comment]
SVG here:
[Icons.svg.zip](https://github.com/naidurubini/empact.cc/files/1769564/Icons.svg.zip)

**Comment by naher94** [comment]
mind if I clean up the icons a little?

**Comment by naidurubini** [comment]
not at all. please feel free to run with your creative expertise on any of this. 

I also wonder how the empathy icon can display immersive storytelling and/or a storytellers and story listeners. Is it possible to show action as many people engaging in what's perhaps even different ways of "action?

**Comment by naher94** [comment]
icon added

**Comment by naidurubini** [comment]
cool. animation still possible?
